GUI/MainForm.cs

```diff
@@ -649,13 +649,10 @@ private void Timer_Tick(object sender, EventArgs e)
             _systemTray.Redraw();
             if (_gadget != null)
                 _gadget.Redraw();
-

```

Comment:
Why are all these lines now removed? Is that a cause of the when_multiline? I'd rather have them back after such a statement (same as in the other files).

Reply:
i deleted this lines, for this short functions i dont need any spacing lines for reading.

Reply:
After brackets or control flow changes should be a new line for readability. I'll make a couple of changes, rest looks good now. Somehow this PR got broken as GitHub doesn't see this branch anymore so I'll recreate it.